src/Compilers/CSharp/Portable/Compilation/CSharpDeterministicKeyBuilder.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal sealed class CSharpDeterministicKeyBuilder : DeterministicKeyBuilder
     {
-        internal static readonly CSharpDeterministicKeyBuilder Instance = new();
+        internal static readonly CSharpDeterministicKeyBuilder Instance = new CSharpDeterministicKeyBuilder();
```

Comment:
This is quite unfortunate. Does the compiler team completely forbids target-typed new? or is it okay for field initializers where the type is already clear and in the same statement?